packages/react-components/src/stories/ToastWrapper.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { ComponentMeta } from '@storybook/react';
+
+import {
+  IToastWrapperProps,
+  ToastWrapper as ToastWrapperComponent,
+} from '../components/ToastWrapper';
+import { Variants } from '../components/Toast';
+
+export default {
+  title: 'Components/Toast Wrapper',
+  component: ToastWrapperComponent,
+} as ComponentMeta<typeof ToastWrapperComponent>;
+
+export const ToastWrapper = (args: IToastWrapperProps): React.ReactElement => (
```

Comment:
Now I am just thinking that maybe we should rename this component. Instead of a generic `wrapper` name maybe some sort of "positioner" would be a better name. 

I am also wondering if this story should be placed under `components`. as far as I remember we discussed that `patterns` would be a better place.

Reply:
Yup, positioning should be described in Patterns. I think positioner is much more obvious than wrapper.